### ISSUE TITLE
Reverse `config-cycle` binding

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#7683]
### Discussed in https://github.com/qutebrowser/qutebrowser/discussions/7667

<div type='discussions-op-text'>

<sup>Originally posted by **mrossinek** April 16, 2023</sup>
I have a key binding like the following in my config:
```python
config.bind("<Ctrl+=>", "config-cycle zoom.default 100% 125% 150% 175% 200%")
```
This allows me to cycle through multiple zoom settings e.g. when switching from a low-res internal to high-res external screen. So far, when switching back, I always need to hit the binding multiple times to "cycle" through when e.g. wanting to go from 150% to 100%. So I was wondering: is there a way to make the reverse mapping of this and have them aligned? E.g. add a binding for `Ctrl + -` which knows the current zoom state and respects the existence of both bindings at the same time?</div>

**Comment by The-Compiler** [comment]
I don't think it should be difficult to add an option which uses `-1` instead of `+ 1` here:

https://github.com/qutebrowser/qutebrowser/blob/346a39ba14bdc05ce21dc1233027803dcaf34cc6/qutebrowser/config/configcommands.py#L236

**Comment by mrossinek** [comment]
I was about to open a PR (see commit linked above) when I realized that this feature is actually not required for me use-case, because qutebrowser actually handles the statefulness of the config setting already. I.e. my exact use-case can be achieved as follows:
```
:bind <Ctrl+=> config-cycle zoom.default 100% 125% 150% 175% 200%
:bind <Ctrl+-> config-cycle zoom.default 200% 175% 150% 125% 100%
```
where repeatedly using `Ctrl+=` or `Ctrl+-` will cycle correctly in the respective directions.

If you think that adding a `--reverse` option can be useful in other scenarios or even just to allow "easier" configuration writing as it avoids having to keep 2 lists in sync (plus reversed), I am happy to actually open the PR to do that. But I guess even that can be done in a Python config by means of simple list reversal.. :thinking:
